en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-apis-as-immutable-gateway.md

```diff
@@ -0,0 +1,63 @@
+# Deploy APIs as immutable gateway
+
+In this case, the APIs will be deployed during the startup. After startup, no APIs can be added using apictl. 
+For that, the apictl projects needs to be mounted to the `docker-compose/resources/adapter/artifacts/apis` directory 
+(default location) of the adapter. 
+
+
+## Step 1 - Download apictl and Set the Path Variable 
+
+First download [apictl](https://github.com/wso2/product-apim-tooling/releases) locally and extract it into a folder of your choice. Then, add its location to your PATH variable.
+
+```
+export PATH=$PATH:<directory-of-the-apictl-executable>
+```
+
+## Step 2 - Create an API Project using apictl
+
+Let's create your first project "petstore" using an open API definition. The following `apictl init` command will generate a project folder containing all the necessary files.
+
+!!! warning
+
+    If you have used a previous version of apictl before, remember to delete the directories `.wso2apictl` and `.wso2apictl.local` that are located in your home directory. Deleting them will make the newer apictl create them again, with content compatible with the current version.
+
+``` bash
+apictl init petstore --oas https://petstore.swagger.io/v2/swagger.json
+```
+
+!!! note
+
+    Instead of initializing the api project, you can [export the apis from api manager]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/migrating-apis-to-different-environments.md) as well.
```

Comment:
```suggestion
    Instead of initializing the API project, you can [export the APIs from API Manager]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/migrating-apis-to-different-environments) as well.
```